vagrant/roles/pxe-server/tasks/main.yml

```diff
@@ -0,0 +1,144 @@
+---
+- name: install required RPMs
+  package:
+    name: "{{item}}"
+    state: present
+  become: yes
+  with_items: "{{required_rpms}}"
+
+- name: create configuration file for eth1
+  template:
+    src: ifcfg-eth1.j2
+    dest: /etc/sysconfig/network-scripts/ifcfg-eth1
+  become: yes
+
+- name: restart network
+  service:
+    name: network
+    state: restarted
+  become: true
+
+- name: modify configuration file of dhcp server
+  template:
+    src: dhcpd.conf.j2
+    dest: /etc/dhcp/dhcpd.conf
+  become: yes
+
+- name: enable dhcpd service
+  service:
+    name: dhcpd
+    enabled: yes
+  become: true
+
+- name: start dhcpd service
+  service:
+    name: dhcpd
+    state: started
+  become: true
+
+- name: enable tftp.socket
+  service:
+    name: tftp.socket
+    enabled: yes
+  become: true
+
+- name: start tftp.socket
+  service:
+    name: tftp.socket
+    state: started
+  become: true
+
+- name: enable vsftpd service
+  service:
+    name: vsftpd
+    enabled: yes
+  become: true
+
+- name: start vsftpd service
+  service:
+    name: vsftpd
+    state: started
+  become: true
+
+- name: download minimal installation iso image to localhost
+  get_url:
+    url: "{{iso_img_url}}"
+    dest: vagrant_data
```

Comment:
Please use the path to the actual file, not the parent dir, otherwise the download still happens! (Did not know this prior).

This worked for me: `dest: "vagrant_data/{{iso_img_file}}"`

Reply:
I use local web server for testing and I didn't notice it. Thanks.